### PR TITLE
Rule SLES-12-010270 extended with ansible, bash and tests

### PR DESCRIPTION
**Pull request by rumch-se**
#### Description:

- _Review required_

#### Rationale:

- _The Rule SLES-12-010270 was extended with ansible, bash and tests. Exchange of codes SLES and CCE between the 2 rules_

- Fixes # _Issue number here (e.g. #26) or remove this line if no issue exists._

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/tests/correct_min_pas_age.pass.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# platform = multi_platform_sle
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
```
Comment (ggbecker) [inline linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/tests/correct_min_pas_age.pass.sh:7]: tests don't make use of this metadata
```suggestion
```

File: linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/tests/incorrect_min_pass_age.fail.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# platform = multi_platform_sle
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
```
Comment (jan-cerny) [inline linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/tests/incorrect_min_pass_age.fail.sh:7]: This doesn't have a meaning in the test scenarios, it's used in the bash remediation instead.

Comment (rumch-se) [reply linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/tests/incorrect_min_pass_age.fail.sh:7]: Thank you @jan-cerny

Comment (jan-cerny) [reply linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/tests/incorrect_min_pass_age.fail.sh:7]: great, but please remove it from here

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**